src/azul/terraform.py

```diff
@@ -793,6 +793,18 @@ def tf_config(self, app_name):
         del deployment['lifecycle']
         deployment['triggers'] = {'redeployment': deployment.pop('stage_description')}
 
+        # Instead of specifying this property in the related Terraform
+        # resource, using an AWS API Gateway extension to the OpenAPI
+        # specification lets us avoid maintaining a complicated trigger
+        # dependency between Terraform resources.
```

Comment:
This will not make sense to anyone not intimately familiar with the issue. I think you shouldn't explain what the solution was (the complicated dependencies) but rather what the problem was (the property reset, and when it occurred).